pkg/client/src/app/pages/applications/components/application-identity-form/field-names.ts

```diff
@@ -0,0 +1,3 @@
+export const APPLICATION_NAME = "applicationName";
+export const MAVEN_SETTINGS = "mavenSettings";
+export const SOURCE_CREDENTIALS = "sourceCredentials";
```

Comment:
Why using those hard coded values for simple strings ?

Reply:
It's just a personal preference since we reuse these strings so often. Less prone to typos & helps with intellisense within vscode.